casts/cards/src/UICollectionCard/CollectionSlide.tsx

```diff
@@ -73,7 +73,7 @@ function CollectionSlideImpl({ content, onLoad, onError, style, isVisible }: Col
     }, [content, contentType, onError]);
 
     const visibility = React.useMemo<StyleProp<ViewStyle>>(() => {
-        return { display: isVisible ? 'flex' : 'none' };
+        return { opacity: isVisible ? 1 : 0 };
```

Comment:
Actually opacity is not the best solution to `hide` things, as on some platforms it disables optimisations. (Like from what I know how it works: usually during rendering, views layers (in terms of iOS) will be turned in textures, and then gpu will try to blend their pixels with regard of their positioning. So, from what I know, if a view is transparent, it's still going to take part in blending, that might load GPU with meaningless work. (more on that https://www.objc.io/issues/3-views/moving-pixels-onto-the-screen/).
So, we may do { top: isVisible ? 0 : -9999, left: isVisible ? 0 : -9999 }, that theoretically better for rendering.
Though I don't know how to measure it, it's just a theory :)

Reply:
Btw, to save on GC calls you can do `isVisible ? styles.notTransparent : styles.transparent` :)

Reply:
That’s actually true, thanks. My original idea was to make it indivisible with display manipulation, but it happened to be not working in Android properly as the card was completely removed from DOM, which was important for us to load the images while being invisible